pyspark/pyproject.toml

```diff
@@ -0,0 +1,94 @@
+# copyright 2022-2023 alibaba group holding limited.
```

Comment:
Ditto.

Reply:
`Pyproject.toml` is the modern way to configure python projects. It contains dependency groups (like `spark`, `tests`, `docs`, etc.) but not only them. In our case this file contains code-style rules (section `[tool.ruff]`), pytest coverafe exclude (section `[tool.coverage.run]`), information about the package name, version, include, etc.

Reply:
You are right, we should remain it. Thank you for letting me learn new knowledge about poetry.